 <launch>
	<node pkg="joy" type="joy_node" name="joy_node">
		<param name="dev" type="string" value="/dev/input/js0"/>
	</node>
<!--
	<node pkg="pr_kpp_test_joycon" type="pr_kpp_joycon_node" name="pr_kpp_joycon_node"> </node>

	<node pkg="pr_kick" name="kick_node" type="kick_node">
		<param name="FREQ"   value="100"/>
		<param name="POW"    value="240"/>
		<param name="DELAY"  value="10"/>
	</node>
-->
	<node pkg="pr_kick" name="new_kick_node" type="new_kick_node">
		<param name="POW"    value="240"/>
		<param name="DEG"    value="-22"/>
		<param name="DELAY_SOL_1" value="100"/>
		<param name="DELAY_SOL_2" value="200"/>
	</node>
	<node pkg="pr_pick_pass" name="new_pick_pass_node" type="new_pick_pass_node">
		<param name="POW_LOWER" value="-70"/>
		<param name="POW_RAISE" value="80"/>
		<param name="POW_WIND" value="240"/>
		<param name="DEG_1" value="-170"/>
		<param name="DEG_2" value="-60"/>
		<param name="DEG_3" value="0"/>
		<param name="DELAY_SOL_1" value="50"/>
		<param name="DELAY_SOL_2" value="450"/>
		<param name="DELAY_HAND" value="500"/>
		<param name="DELAY_LAUNCH" value="800"/>
		<param name="DELAY_WIND" value="150"/>
	</node>

	<node pkg="rosserial_python" type="serial_node.py" name="serial_node1">
		<param name="port" value="/dev/ttyACM0"/>
	</node>

<!--	<node pkg="rosserial_python" type="serial_node.py" name="serial_node2">
		<param name="port" value="/dev/ttyACM2"/>
	</node>
-->

	<!-- ashimawari -->
<!--	<rosparam command="load" file="$(find pr)/constant.yaml"/>
	<rosparam command="load" file="$(find pr)/state.yaml"/>

	<arg name="arduino_port" value="/dev/ttyACM0"/> 

	<param name="/joy_node/dev" value="/dev/input/js0"/>

	<node pkg="rosserial_python" name="serial_node"
		type="serial_node.py" args="$(arg arduino_port)"/> 
	<node pkg="joy" name="joy" type="joy_node"/>

	<node pkg="pr" name="task_manager" type="pr_task_manager.py"/>
	<node pkg="pr" name="arduino_driver" type="pr_arduino_driver.py"/>
-->
</launch>

